packages/phone-number-privacy/signer/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "dependencies": {
     "@celo/contractkit": "0.4.5",
-    "@celo/phone-number-privacy-common": "1.0.6",
+    "@celo/phone-number-privacy-common": "1.0.7-dev",
```

Comment:
Have you confirmed that the docker image still builds correctly when you use a dev version here?

Reply:
Actually I know that it will _not_ build successfully haha. I'll change these versions once I've published the common package